rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
+|    amd64     |   Tier 1 [d][a][s]    |    Tier 1 [a][s]     |       Tier 3 [s]        |     Tier 3 [s]     |                |
++--------------+-----------------------+----------------------+-------------------------+--------------------+----------------+
+|    arm64     |   Tier 1 [d][a][s]    |                      |                         |     Tier 3 [s]     |   Tier 3 [s]   |
++--------------+-----------------------+----------------------+-------------------------+--------------------+----------------+
+|    arm32     |      Tier 3 [s]       |                      |                         |     Tier 3 [s]     |   Tier 3 [s]   |
++--------------+-----------------------+----------------------+-------------------------+--------------------+----------------+
+
+The following indicators show what delivery mechanisms are available for each platform.
+
+" [d] " Debian packages will be provided for this platform for packages submitted to the rosdistro.
+
+" [a] " Binary releases are provided as a single archive per platform containing all packages in the Galactic ROS 2 repos file [10]_.
+
+" [s] " Compilation from source.
+
+Middleware Implementation Support:
+
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                  |
++==========================+=====================+===============+=============================+================================+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS   | Tier 1        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_connext_cpp          | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64 |
+|                          |                     |               | and OpenEmbedded            |                                |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS   | Tier 2        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.7
```

Comment:
I forget; is this driven by Debian Buster?

Reply:
I think so, but we actually discussed relaxing this to 3.6 to include CentOS 7 and 8 a few weeks ago. Thoughts on that?

Reply:
Oh right, I totally forgot about that discussion.  I'm fine with that too.

Reply:
9ad7a1f